### PR TITLE
added bongocat keymap to 0xCB Static

### PR DESCRIPTION
**Pull request by Conor-Burns**
## Description
Added bongocat animation with WPM counter.
Enabled by pressing ESC on layer 4 (toggles)

## Types of Changes

- [x] Keymap/layout/userspace (addition or update)

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.

- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).

- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/0xcb/static/keymaps/bongocat/keymap.c [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+// clang-format off
+enum my_keycodes {
+  WPM
```
Comment (drashna) [inline keyboards/0xcb/static/keymaps/bongocat/keymap.c:20]:
```suggestion
    WPM = SAFE_RANGE,
```

File: keyboards/0xcb/static/keymaps/bongocat/keymap.c [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+        }
+        return false;
+    default:
+        return true;
+  }
+}
+/*layer switcher */
+layer_state_t layer_state_set_user(layer_state_t state) {
+state = update_tri_layer_state(state, _FN2, _FN3, _FN4);
+return state;
```
Comment (drashna) [inline keyboards/0xcb/static/keymaps/bongocat/keymap.c:75]:
```suggestion
    state = update_tri_layer_state(state, _FN2, _FN3, _FN4);
    return state;
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
Also, it may be a good idea to run a format pass on these files. 
```sh
qmk cformat keyboards/0xcb/static/keymaps/bongocat/*
```

You may want to wrap the keymap itself and the animation chars with  `// clang-format off` and `// clang-format on` so it doesn't reformat those.

**Review by drashna — APPROVED** [review]

**Comment by Conor-Burns** [comment]
> Also, it may be a good idea to run a format pass on these files.
> 
> ```shell
> qmk cformat keyboards/0xcb/static/keymaps/bongocat/*
> ```
> 
> You may want to wrap the keymap itself and the animation chars with `// clang-format off` and `// clang-format on` so it doesn't reformat those.

Ok thank you!

**Review by fauxpark — APPROVED** [review]

**Pull request merged**